README.md

```diff
@@ -1,2 +1,50 @@
-# folio-deployment
-Deploy Folio to our Infrastructure
+# okube 
+An Okapi deployment toolkit.
+
+Automation for deploying a Okapi instance and its modules to a Kubernetes (k8s) cluster from a command line interface. The easy button is here.
+
+
+## Why okube?
+
+Successfully standing up Okapi and its modules into a cluster requires that certain things that we are calling 'task' happen in a specific order. Doing so manually requires that the opertator remembers the list of task and their execution order which can be error-proned. In addition, this process can take a considerable amount of time as the operator needs to wait for a task to complete before proceeding to the next. Unfortuately task execution order is not the only knowledge needed for a deployment. 
+
+Kubernetes presents terminology, tools, and techniques specific to Kubernetes that need to be learned before deploying to a cluster. This can be a tall order as Kubernetes and Devops in general cover a large surface area. Fortuately this `okube` toolkit eleviates the need to learn Okapi task execution order, Kubernetes, and Devops straight out the gate.
+
+Deploying Okapi and the modules to a new Kubernetes cluster just takes running a command such as `bin/okube deploy --environment sandbox`.
+
+## Pre-deploy Caveat
+
+Currently this toolkit is hard coded for use with Frontside's Google Container Engine (GKE) instances but can be tailored for use with another GKE instance wanting to deploy Okapi and the modules.
+
+## Deploying Cluster
+To run any commands you can `cd` (change directory) in the `scripts` folder from the root of these repo.
+
+Commands available:
+ * `bin/okube deploy --enviroment sandbox`
+ Deploys a cluster to the development environment.
+ * `bin/okube deploy --enviroment production`
+ Deploys a cluster to the production environment.
+
+Okube in the process of deploying to a Kubernetes cluster will automate several task for you such as:
+* Deploy Ingress
+* Deploy Nginx Controllers + Default Backend
+* Configure + Update DNS Record
+* Deploy + Configure Kube Lego
+* Verify TLS Certificate
+* Deploy Okapi
+* Create a Tenant
+* Create List of Desired Modules based on list in `folio.conf`
+* Resolve Dependencies of modules from the desired modules and the availble modules
+* Deploy + Register resolved modules with Okapi
+* Discover modules
+* Enable modules for tenant
+* Create an 'admin' User
+
+
+## Destroying Cluster
+
+!Imporant before running this command make sure you are in the correct cluster known as `context`!
+
+To view you current current context you can run `kubectl config current-context` from the command line.
+
+* `bin/okube destroy`
```

Comment:
It might even be worth having them type in the name of the cluster as confirmation. Perhaps with a `--destroy-no-confirm` option to avoid a prompting for cases when doing it as part of some automation.